### DISCUSSION TITLE
API to get keyboard mapping map

### DISCUSSION THREAD

**Discussion by rahmanusta** [ajaxorg/ace#5034] [Q&A]
Hi there,

Is there any API which provides all default and user overrided keybindings and targeted action as a JS Object like below ?

```
let keyMapping= editor.getKeyBindings();
 {
      name: "Copy", win: "Ctrl+N", mac : "Command+N", action : () => { ..... },
      ......,
      ......
 }
```

**Comment by InspiredGuy** [comment]
Hey, there is a limited support for this now with https://github.com/ajaxorg/ace/blob/master/src/ext/menu_tools/get_editor_keyboard_shortcuts.js
